aries/aries_vcx_core/src/wallet/indy/mod.rs

```diff
@@ -83,11 +83,14 @@ impl Drop for IndyWalletRecordIterator {
     }
 }
 
-#[derive(Clone, Debug, Default, Builder, Serialize, Deserialize)]
-#[builder(setter(into, strip_option), default)]
+#[derive(Clone, Debug, Default, TypedBuilder, Serialize, Deserialize)]
+#[builder(field_defaults(setter(into, strip_option), default))]
 pub struct WalletConfig {
+    #[builder(setter(!strip_option))]
```

Comment:
Since more fields are non-option than options, it would be cleaner to simply define `strip_option` on the field which use `Option`, rather than defining `strip_option` as default and disabling it on the fields it doesn't apply on.

Reply:
Changed.